Cubical/Foundations/Embedding.agda

```diff
@@ -0,0 +1,81 @@
+{-# OPTIONS --cubical --safe #-}
+
+module Cubical.Foundations.Embedding where
+
+open import Cubical.Core.Everything
+
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Transport
+open import Cubical.Foundations.Isomorphism
+
+private
+  variable
+    ℓ : Level
+    A B : Set ℓ
+    f : A → B
+    w x : A
+    y z : B
+
+isEmbedding : (A → B) → Set _
+isEmbedding f = ∀ w x → isEquiv {A = w ≡ x} (cong f)
```

Comment:
Can you please add the easy lemma that `isEmbedding` is a proposition?

Reply:
This has been proved.